dts/bindings/dma/st,stm32-dma-v2.yaml

```diff
@@ -17,35 +17,62 @@ description: |
     this value is 0 for Memory-to-memory transfers
     or a value between <1> .. <dma-generators> (not supported yet)
     or a value beweeen <dma-generators>+1  ..  <dma-generators>+<dma-requests>
-    3. channel-config: A 32bit mask specifying the DMA channel configuration
```

Comment:
Please keep this, it is still valuable, clear and readable information.

Reply:
OK